src/Storage/Models/InstanceQueryParameters.cs

```diff
@@ -348,26 +358,38 @@ private static void AddDateParamIfNotNull(Dictionary<string, object> postgresPar
             }
         }
 
+        private bool? GetConfirmed()
+        {
+            if (Confirmed.HasValue)
+            {
+                return Confirmed.Value;
+            }
+
+            return ExcludeConfirmedBy.HasValue && ExcludeConfirmedBy.Contains(Org) ? false : null;
+        }
+
         /// <summary>
-        /// Retrieves an array of exclude confirmed by values from the query values.
+        /// Retrieves an array of exclude confirmed by values that differ from the invoking org.
```

Comment:
"Invoking org" can be a bit misleading. Remember that these parameters can be used by end user systems as well. The Org parameter is simply an instance filter like any other.